cilium/cmd/ip_metadata.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/ipcache"
+)
+
+var ipMetadataCmd = &cobra.Command{
+	Use:     "metadata",
+	Aliases: []string{"md"},
```

Comment:
I don't think it's necessary to add an alias for this command.

Reply:
removed the alias